server/webapp/WEB-INF/rails/webpack/views/pages/pipelines/scm_material_fields.tsx

```diff
@@ -44,9 +46,13 @@ abstract class ScmFields extends MithrilViewComponent<Attrs> {
 
   view(vnode: m.Vnode<Attrs>): m.Children {
     const mattrs = vnode.attrs.material.attributes() as ScmMaterialAttributes;
+    let materialTest = <TestConnection material={vnode.attrs.material}/>;
```

Comment:
Minor nitpick, but maybe either:

- Use ternary, or
- `if`/`else`
- also, extract as function?

The minor benefit of using ternary or if/else is (I think, anyway) they are special forms where only 1 condition is evaluated. That way you are not potentially building both `<TestConnection/>` and `<MaterialCheck/>` and instead just one of them. In the big picture, this is a micro-optimization, but not a bad one IMO.